tests/tests.rs

```diff
@@ -124,45 +124,37 @@ fn test_invalid_muxed_ed25519() {
     // assert_eq!(r, Err(DecodeError::Invalid));
 
     // Invalid length (congruent to 6 mod 8)
-    let r = Strkey::from_string(
-        "MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVAAAAAAAAAAAAAJLKA",
-    );
+    let mut r: Result<Strkey, _>;
+
+    r = "MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVAAAAAAAAAAAAAJLKA".parse();
     assert_eq!(r, Err(DecodeError::Invalid));
 
     // Invalid length (base-32 decoding should yield 43 bytes, not 44)
-    let r = Strkey::from_string(
-        "MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVAAAAAAAAAAAAAAV75I",
-    );
+    r = "MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVAAAAAAAAAAAAAAV75I".parse();
```

Comment:
Why is a `mut`able value preferred in these tests? It doesn't appear to need to be mutable.